addons/web/static/src/core/emoji_picker/emoji_picker.xml

```diff
@@ -73,7 +73,7 @@
 </t>
 
 <t t-name="web.EmojiPicker.tab">
-    <span class="o-Emoji text-center fs-5 rounded-3 cursor-pointer d-flex align-items-center align-self-stretch" t-att-class="{'o-active': category.sortId === state.categoryId}" t-att-title="category.name" t-att-data-id="category.sortId" t-on-click="() => this.selectCategory(category.sortId)">
+    <span class="o-Emoji text-center fs-5 rounded-3 cursor-pointer d-flex align-items-center align-self-stretch" tabindex="0" t-att-class="{'o-active': category.sortId === state.categoryId}" t-att-title="category.name" t-att-data-id="category.sortId" t-on-click="() => this.selectCategory(category.sortId)">
```

Comment:
Adding `tabindex="0"` on tab can be useful, e.g. use `Tab` to select a category from focus on search.
However there are missing stuffs for this addition to make sense:
- `Enter` when a category is focused should behave like a click on the category
- click on category in navbar should put focus on search rather than navbar, otherwise the keyboard navigation would highlight this category item and the active category, which is confusing (the fomer should not be there)
![Screenshot 2025-01-28 at 11 35 14](https://github.com/user-attachments/assets/34a0ebc7-b2ff-4489-aa1d-8cd398a7388b)
- clicking on category should focus the 1st item of the category, so that the keyboard navigation makes sense

Reply:
I have done the 2 first points.
Selecting the first emoji of a category doesn't look possible with the current tools :shrug:

Reply:
Done